apps/dashboard/src/main/resources/inbuilt_test_yaml_files/BasicCSRF.yaml

```diff
@@ -0,0 +1,54 @@
+id: BASIC_CSRF
+info:
+  name: "BASIC_CSRF"
+  description: "BASIC_CSRF"
+  details: >
+            "BASIC_CSRF"
+  impact: "Broken User authentication is a serious vulnerability. Attackers can gain control to other users’ accounts in the system, read their personal data, and perform sensitive actions on their behalf, like money transactions and sending personal messages."
+  category:
+    name: NO_AUTH
+    shortName: Broken Authentication
+    displayName: Broken User Authentication (BUA)
+  subCategory: BASIC_CSRF
+  severity: HIGH
+  tags:
+    - Business logic
+    - OWASP top 10
+    - HackerOne top 10
+  references:
+    - "https://owasp.org/www-project-web-security-testing-guide/v42/4-Web_Application_Security_Testing/"
+    - "https://github.com/OWASP/API-Security/blob/master/2019/en/src/0xa2-broken-user-authentication.md"
+    - "https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html"
+    - "https://cwe.mitre.org/data/definitions/798.html"
+
+api_selection_filters:
+  response_code:
+    gte: 200
+    lt: 300
+  method:
+    contains_either:
+      - "POST"
+      - "PUT"
+      - "PATCH"
+      - "DELETE"
+  request_headers:
+    for_one:
+      key:
+        regex: ^(?!.*host)(?!.*cookie)(?!.*content-type).*$
+        extract: header_key
+
+execute:
+  type: single
+  requests:
+    - req:
+      - delete_header: ${header_key}
```

Comment:
FYI: This will just delete 1 header for now

Reply:
We will have to add functionality for this